functional_tests/basic_java_test/hello_world/hello.java

```diff
@@ -0,0 +1,27 @@
+package com.sleepingcat;
```

Comment:
I appreciate the company name here, but I think I'd prefer com.csbuild - that way we're not tying csbuild to sleepingcat in case we ever want to move it to a different ownership.

Also this file is missing the license header. As are several other files (including files that are checked in already). I'm going to add .java, .c, and .cpp files to our license header check. Not something you have to do for this review, I'll do it.